go/vt/vttablet/tabletmanager/rpc_backup.go

```diff
@@ -27,6 +27,7 @@ import (
 	"vitess.io/vitess/go/vt/topo/topoproto"
 	"vitess.io/vitess/go/vt/vterrors"
 
+	"vitess.io/vitess/go/vt/proto/tabletmanagerdata"
```

Comment:
can you change this to match the other aliased proto imports? (should just append `pb`)